tools/projmgr/docs/Template/MyProject/.clangd

```diff
@@ -0,0 +1,3 @@
+CompileFlags:
```

Comment:
this file reference contains your personal file location. Is this useful to be checked in?